### ISSUE TITLE
Have colour coded feedback when a test fails or passes

### ISSUE THREAD

**Issue by james-ashman** [galasa-dev/projectmanagement#1014]
It would be a really nice feature to be able to visually see if a test has failed or passed before clicking into the run itself.

**Comment by Michael-Baylis** [comment]
Yep,  and with icons,   but unfortunately it is very low priority
